test/integration/kubernetes_deploy_test.rb

```diff
@@ -917,6 +919,19 @@ def test_cronjobs_can_be_deployed
     cronjobs.assert_cronjob_present("my-cronjob")
   end
 
+  def test_jobs_can_be_successful
+    assert_deploy_success(deploy_fixtures("jobs"))
+  end
+
+  def test_jobs_can_fail
+    fixtures = deploy_fixtures("jobs") do |f|
```

Comment:
- Same comment about using the fixture from the hello-cloud set instead. 
- Can you run this a whole bunch of times in a row to confirm it isn't flakey? I'm worried that the deploy will race that first pod crash. If that's the case, we could deploy it with something that is guaranteed to take longer (e.g. a deployment that hits an annotation-based timeout).